.factory/automation.yml

```diff
@@ -170,6 +170,17 @@ build:
         bazel run @typedb_dependencies//tool/bazelinstall:remote_cache_setup.sh
         bazel test $(bazel query 'kind(rust_test, //tests/behaviour/service/...)') --test_output=streamed --jobs=1
 
+    test-crash-resilience:
+      image: typedb-ubuntu-22.04
+      command: |
+        sudo apt update
+        sudo apt install -y libclang-dev
+        bazel run @typedb_dependencies//tool/bazelinstall:remote_cache_setup.sh
+        bazel build //:assemble-docker-x86_64.tar //:assemble-all-linux-x86_64-targz
+        tar -xf bazel-bin/typedb-all-linux-x86_64.tar.gz
+        docker load -i bazel-bin/assemble-docker-x86_64.tar
+        tool/test/simulate-crash.sh
```

Comment:
won't this still block release if its in the normal corectness workflow? i dunno if alex/ganesh remembers how to put it in a different one

Reply:
Yep. Unfortunately, Factory hardcodes that the only two pipelines we can have are `build` and `release`, and as soon as any workflow in build fails, the whole pipeline fails.

Reply:
Well, more pointers to start moving off factory if poss